internal/learn/the-community/qanda.md

```diff
@@ -0,0 +1,98 @@
+
+Btw.: All companies have to document and tax value transactions. So all data transactions have to be recorded in blockchains for accounting purposes. With IOTA, not only the fees for data transactions are eliminated but also the associated accounting including Tx storage (up to 10 years) for the majority of all transactions.
+
+**Chain of events**
+Only with a system in which data transactions can be linked directly to value transactions the complete event chain of the data trade can be correctly traced. The IOTA token reflects the (potential) value generated by the data transaction. Therefore, it would make no sense at all if only the IOTA data transactions were used but the payment of this data takes place outside the IOTA protocol.
+
+Payment in the same system is very advantageous for all parties involved, for example, if it is done in a digital decentralized marketplace. Let’s take a look at the current process for placing an order. As a rule, an order always starts a series of events that are based on each other. This could look like this: Order > Verification > Confirmation > Payment > Check receipt of payment > Processing and shipping. This process can of course be solved individually or partially automated but basically this chain of events is always the same.
+
+The IOTA technology with the digital marketplaces enables a free, secure and trustless linking of unknown business partners. The technology offers a very cost effective or even free possibility to link the chain of events of an order in a single system at the correct time. If one were to outsource individual events, e.g. external payment without IOTA, it would be an additional effort to enter this individual event back into the event chain of the marketplace at the correct time. This would cause additional costs compared to the pure IOTA solution and the IOTA solution would therefore have competitive advantages.
+
+**Cross-country and cross-company exchange medium**
+If you want to map cross-country and cross-company processes, trade or logistics between machines these transactions can take place without much delay and without having to agree on a trading currency etc. The only requirement is that machines have direct access to a stock exchange either directly or via a financial service provider to make conversions. This could be, for example, the house bank of a company, which keeps the Fiat accounts anyway. If the bank charges a small margin for this autonomous conversion job this is absolutely legitimate. A machine could autonomously submit a conversion order to the bank at any time and then have its IOTA tokens available to send them across the national border. In this case, the higher volatility is completely neglected and a machine would not have to keep tokens in its wallet permanently.
```

Comment:
```suggestion
If you want to map cross-country and cross-company processes, trade, or logistics between machines these transactions can take place without much delay and without having to agree on a trading currency, etc. The only requirement is that machines have direct access to a stock exchange directly or via a financial service provider to make conversions. This could be, for example, the house bank of a company, which keeps the Fiat accounts anyway. If the bank charges a small margin for this autonomous conversion job, this is absolutely legitimate. A machine could autonomously submit a conversion order to the bank at any time and then have its IOTA tokens available to send across the national border. In this case, the higher volatility is completely neglected, and a machine would not have to permanently keep tokens in its wallet.
```